cloudsync/providers/dropbox.py

```diff
@@ -153,6 +153,8 @@ def interrupt_auth(self):
         self._flow = None
 
     def _on_oauth_success(self, auth_dict):
+        # dropbox doesn't support this
+        auth_dict.pop("expires_in", None)
```

Comment:
This should be fixed if you're running the latest version of the dropbox sdk. Is your environment up to date?